zio-query/shared/src/main/scala/zio/query/CompletedRequestMap.scala

```diff
@@ -133,14 +189,34 @@ object CompletedRequestMap {
     map
   }
 
-  private[query] object unsafe {
+  /**
+   * Constructs a completed requests map containing a single entry
+   */
+  def single[E, A](request: Request[E, A], exit: Exit[E, A]): CompletedRequestMap =
```

Comment:
How about an `apply` constructor with one or more values?

Reply:
Is this what you mean?

```scala
def apply[E, A](results: (Request[E, A], Exit[E, A])*): CompletedRequestMap
```

Reply:
Yep